custom_components/myskoda/sensor.py

```diff
@@ -169,6 +169,9 @@ def native_value(self) -> float | None:  # noqa: D102
         if status := self._status():
             return status.charge_power_in_kw
 
+    def required_capabilities(self) -> list[CapabilityId]:
+        return [CapabilityId.CHARGING_MEB]
+
```

Comment:
If you only know they are not present in one type of vehicle, it would be better to use `forbidden_capabilities` to make sure the sensor does not show up for your type of car. I think it's `CHARGING_MQB`